dkg-gadget/src/async_protocols/blockchain_interface.rs

```diff
@@ -202,6 +202,7 @@ where
 		key: LocalKey<Secp256k1>,
 		session_id: SessionId,
 	) -> Result<(), DKGError> {
+		log::info!(target: "dkg", "Storing public key {:?} for session {:?}", key, session_id);
```

Comment:
This a bit redundant, I would say a better way is to keep all these logs inside the database backend.